app/index.html

```diff
@@ -1,17 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <title>Interceptor</title>
   <meta name="viewport" content="width=device-width">
   <meta name="description" content="" />
   <meta name="keywords" content="" />
   <link href="https://fonts.googleapis.com/css?family=Barlow:400,600" rel="stylesheet" />
```

Comment:
DO we still need this font?

Reply:
Yeah, the font Anonymous Pro is used only inside the textarea where json/any response is shown.. Other places, the font `Barlow` looks good